Mathlib/CategoryTheory/Bicategory/NaturalTransformation/Oplax.lean

```diff
@@ -164,4 +179,154 @@ instance : CategoryStruct (OplaxFunctor B C) where
 
 end OplaxTrans
 
+/-- A strong natural transformation between oplax functors `F` and `G` is a natural transformation
+that is "natural up to 2-isomorphisms".
+
+More precisely, it consists of the following:
+* a 1-morphism `η.app a : F.obj a ⟶ G.obj a` for each object `a : B`.
+* a 2-isomorphism `η.naturality f : F.map f ≫ app b ⟶ app a ≫ G.map f` for each 1-morphism
+`f : a ⟶ b`.
+* These 2-isomorphisms satisfy the naturality condition, and preserve the identities and the
+  compositions modulo some adjustments of domains and codomains of 2-morphisms.
+-/
+structure StrongOplaxTrans (F G : OplaxFunctor B C) where
```

Comment:
For consistency with the added `StrongTrans` for pseudofunctors, how about now simply calling this `StrongTrans`? Being oplax (of F and G) is included in the namespace.

Reply:
See https://github.com/leanprover-community/mathlib4/pull/18252#discussion_r1818094153

Reply:
See https://github.com/leanprover-community/mathlib4/pull/18252#discussion_r1818094153

Reply:
`Trans` suggested there is not sufficient, but `StrongTrans` is sufficient. There are 9 possibilities {lax, oplax, pseudo}^2, and in this case (oplax, pseudo) is specified (strong is used to mean pseudo here).

Reply:
I think this is just a mistake, thanks! Should be `StrongTrans` here